src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -295,9 +296,22 @@ export class StudyViewPageStore {
         this.reactionDisposers.push(reaction(() => this.loadingInitialDataForSummaryTab, () => {
             if (!this.loadingInitialDataForSummaryTab) {
                 this.updateChartStats();
+                this.loadUserSettings();
```

Comment:
loadUserSettings should be done before updateChartStatus

Reply:
no, since `updateChartStats` updates chart visibility and type which might be different from user saved ones